bundles/GoogleMarketingBundle/public/css/icons.css

```
@@ -0,0 +1,8 @@
+
+.pimcore_icon_analytics {
+    background: url(/bundles/pimcoreadmin/img/flat-color-icons/pie_chart.svg) center center no-repeat !important;

```

```suggestion
    background: url(/bundles/pimcoreadmin/img/flat-color-icons/pie_chart.svg) center center no-repeat !important;
```
potentially should be copied over in the bundle from `bundles/pimcoreadmin` to be not affected by the changes in core

---

But then we should have done this for all bundles. I think it is okay to keep all the icon images in the admin and just use them in the bundles.

---

let's keep them for now!